scvi/dataset/synthetic.py

```diff
@@ -36,6 +40,14 @@ def __init__(
         # clear potentially unused cell_types
         self.remap_categorical_attributes()
 
+        protein_data = CellMeasurement(
```

Comment:
I updated this so that protein data is independent of gene data. I think it should be fine generating protein data by default right? This doesn't effect at all and scVI functions - it's just another dataset attribute.